dev/com.ibm.websphere.appserver.features/visibility/public/appSecurity-4.0/io.openliberty.appSecurity-4.0.feature

```diff
@@ -0,0 +1,28 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.appSecurity-4.0
+visibility=public
+IBM-API-Package: jakarta.security.enterprise; type="spec", \
+ jakarta.security.enterprise.authentication.mechanism.http; type="spec", \
+ jakarta.security.enterprise.credential; type="spec", \
+ jakarta.security.enterprise.identitystore; type="spec", \
+ jakarta.security.auth.message; type="spec", \
+ jakarta.security.auth.message.callback; type="spec", \
+ jakarta.security.auth.message.config; type="spec", \
+ jakarta.security.auth.message.module; type="spec"
+
+IBM-ShortName: appSecurity-4.0
+Subsystem-Name: Application Security 4.0
+-features=io.openliberty.cdi-3.0, \
+ com.ibm.websphere.appserver.el-4.0, \
+ io.openliberty.security-2.0, \
+ com.ibm.websphere.appserver.servlet-5.0
+-bundles=io.openliberty.jakarta.security.2.0; location:=dev/api/spec/; mavenCoordinates="jakarta.security.enterprise:jakarta.security.enterprise-api:2.0.0-RC2", \
+ io.openliberty.security.jakartasec.2.0.internal, \
+ io.openliberty.security.jakartasec.2.0.cdi.internal, \
+ io.openliberty.security.authentication.tai.internal, \
```

Comment:
I think these should be internal.cdi and internal.tai

Reply:
I have renamed them.